simulator/src/commands/start.ts

```diff
@@ -58,8 +61,20 @@ export abstract class Start extends Command {
     // If you want logs to persist, i.e. via `simulator start 1 2>&1 | tee ~/i/logs/run_1.log` you will
     // need to remove the spinner
     CliUx.ux.action.start(`running simulation ${simName}`)
-    await simulation.run(logger, { persist, duration })
-    CliUx.ux.action.start(`stop simulation ${simName}`)
-    this.exit()
+
+    // The simulator is created here because oclif catches errors so we can't throw them
+    // and handle `uncaughtException` in the simulator. Having this try-catch block is a workaround
+    // to ensure the simulator gracefully exits when an error occurs.
+    Start.simulator = new Simulator(logger, { persist, duration })
```

Comment:
Do we need to set this to `Start.simulator`? Could this just be a normal const?

Reply:
no, made a normal const